ee/server/broker.ts

```diff
@@ -249,90 +254,93 @@ const {
 	SKIP_PROCESS_EVENT_REGISTRATION = 'true',
 } = process.env;
 
-const network = new ServiceBroker({
-	// TODO: Reevaluate, without this setting it was preventing the process to stop
-	skipProcessEventRegistration: SKIP_PROCESS_EVENT_REGISTRATION === 'true',
-	transporter: TRANSPORTER,
-	metrics: {
-		enabled: MS_METRICS === 'true',
-		reporter: [{
-			type: 'Prometheus',
+// only instantiate ServiceBroker and NetworkBroker if has the proper license
+onLicense('scalability', () => {
```

Comment:
Other services should not have this dependency

Reply:
you're right.. I forgot other services were using this file as well